data/praduman8435.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "Praduman Prajapati",
+  "bio": "A **MERN** stack and **DevOps** enthusiast"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Praduman___",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/___.pradyumn.___/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/praduman-prajapati/",
+      "icon": "FaLinkedIn"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/praduman8435",
+      "icon": "FaGithub"
+    },
```

Also, remove this trailing comma `,` here. 

```suggestion
    }
```